Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -1,5 +1,6 @@
 #define LOG_MODULE PcapLogModuleLiveDevice
 
+#include "IpUtils.h"

```

Comment:
Why adding this include?

Reply:
@seladb Fighting with `-std` bugs on MinGw env